pages/dkp/konvoy/2.1/choose-infrastructure/aws/quick-start-aws/index.md

```diff
@@ -42,6 +42,8 @@ Before starting the Konvoy installation, verify that you have:
 
 ## Create a new AWS Kubernetes cluster
 
+If you use these instructions to create a cluster on AWS using the DKP default settings without any edits to configuration files or additional flags, your cluster is deployed on a [CentOS 7 operating system image][supported-systems] with 3 control plane nodes, and 4 worker nodes.
```

Comment:
So what I miss here is what does that mean?. Its good to know for the user but what about

> Using default images will work but due to missing optimizations the created cluster will have certain limits. We strongly suggest using [Konvoy Image Builder to create an custom AMI](../../image-builder/create-ami/).

Reply:
We could add something that says 'use the advanced topics to take advantage of KIB'.

Reply:
Maybe I'm wrong but AFAIK we really want customers to use KIB based images rather than the CAPI ones which would be the default. So I think we should at least state it.

Reply:
I have the same concerns as @fatz - this is way too vague...  See above for my suggested rewording about 'default images'